### PR TITLE
Project: Smashing Magazine

### PR DESCRIPTION
**Pull request by OlukaDenis**
Features:
1. Aligning of elements using grid and flex layouts
2. Application of background shades
3. Using the backgrounds according to the visual hierarchy principles

### CODE REVIEW

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @OlukaDenis and @Grifo89   🌟

**Good work on this project, but we need to fix the issues below before moving on ⬇️** 

1. The `<footer>` needs to have centered links.

2. Please use the `<main>` semantic HTML tag for the main content of the website.


### _After making the required changes please submit a new code review request_

### Good luck guys! 🍀

**Comment by Grifo89** [comment]
Hey @Redvanisation, please just because you can't find errors doesn't mean that you must invent them, We were following some examples sent by students in the Odin Project, and actually It isn't a 
request to have centered links and less to use main tag here is an example https://bojozahariev.github.io/SmashingDesignTeardown/.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        <h2>Smashing Newsletter</h2>
+        <img
+          src="https://d33wubrfki0l68.cloudfront.net/ce7741e1659c0366fda51555365194e0bfd136c0/ae578/images/smashing-cat/cat-with-trampet.svg"
+          alt="image"
+        />
+      </div>
+    </div>
+    <footer class=" full-width footer-container grey-black">
+      <h2>Browse All Smashing Magazine Topics</h2>
+    </footer>
```
Comment (whiz25) [inline index.html]: Your footer should have **centered links**. Please make them visible by adding some dummy text like ```link-1``` or anything you like. This **[centered links]** is a requirement for the completion of this project.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Amazing! :+1: 
This project looks too good. Am impressed.
You have met most of the requirements which is a good thing. However, you have only one thing to make this project complete. Check my comment and implement the requested change.
### [REMINDER]
- The correct use of ```git flow``` is highly emphasized. I know you followed the proper use of git-flow but I believe you can do better. Every feature should have its own branch. For example, the footer, navbar, should all be branches.

#### Make the requested change and re-submit for another review.

**Comment by Grifo89** [comment]
We have done with the centered links. Please check it out.

**Review by whiz25 — CHANGES_REQUESTED** [review]
### You haven't made any changes! :disappointed: 
Make those changes and re-submit.

**Comment by Grifo89** [comment]
The changes are already there. 7910e501e492a44b1d32649b1cf204b6c5d1d1a2

**Review by uche-inyama — APPROVED** [review]
Good job team! 

I want to apologize on behalf of my colleague, you used the **right flow**, **```github flow```**

Go head and merge and master branch

#### **```Have a Great day```** :slightly_smiling_face:

**Pull request merged**